### PR TITLE
Portfolio works Section 

### PR DESCRIPTION
**Pull request by mbonabucya**
Add Works section
Add Images of the work section 
Add Tonic Project details
Add Muti-post stories Project
Add Professional Arts
Add About my self section
Add Languages list
Add Frameworks list
Add Skills

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -76,6 +76,289 @@ <h1>
...
+                <li>
+                    javascript
+                </li>
+            </ul>
+        </div>
+        <div class="Projects">
+            <button type="button">
+                See Project
+            </button>
+        </div>
```
Comment (muneebulrehman) [inline html/index.html]: Please consider sticking to the Figma template of your choosing. Kindly make relevant changes.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

Comment (muneebulrehman) [inline html/index.html]: Also please consider wrapping each project in a `<div>` or preferably an `<article>`

File: html/index.html [hunk 2]
```diff
@@ -76,6 +76,289 @@ <h1>
...
+      <li>
+          javascript
+      </li>
+  </ul>
+</div>
+<div class="Projects">
+  <button type="button">
+      See Project
+  </button>
+</div>
```
Comment (muneebulrehman) [inline html/index.html]: Please fix the indentation-related issues. I see a lot of them.

File: html/index.html [hunk 3]
```diff
@@ -76,6 +76,289 @@ <h1>
...
+                <h1>Frameworks</h1>
+                <img src="../Image/right-arrow.png" alt="">
+            </div>
+            <div class="skills">
+                <h1>Skills</h1>
+                <img src="../Image/right-arrow.png" alt="">
+            </div>
+
+
+           </div>
```
Comment (muneebulrehman) [inline html/index.html]: How about wrapping all the technology categories in a `<ul>` tag and each category in a `<li>` tag

File: html/index.html [hunk 4]
```diff
@@ -76,6 +76,289 @@ <h1>
...
+                   <img src="../Image/dropdown.png" alt=" drop down icon" />
+               </div>
+               <div class="front-end">
+                   <img src="../Image/Ellipse-js.png" alt="js">
+                   <p>JavaScript</p>
+               </div>
+               <div class="front-end">
+                <img src="../Image/Ellipse-html.png" alt="js">
+                <p>Html</p>
+            </div>
```
Comment (muneebulrehman) [inline html/index.html]: Consider making each category a list with individual items as an item list. eg.,
```
<ul>
       <h1>Languages</h1>
       <li>javascript</li>
</ul>
```

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @mbonabucya, 

Good job so far! I can see you put a lot of effort into this section of the project. However, there are some issues that you still need to work on to go to the next project but you are almost there!
Consider this review as a learning opportunity and improve on those coding skills.

## Highlights
- Good documentation
- Good PR description
- Used semantic HTML tags.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- IMO, you are overusing the `<div>` tag.  You may consider doing a reading of [this ](https://www.w3schools.com/html/html_blocks.asp) article which talks about differences between block and inline elements in HTML.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: html/index.html (hunk 1 above)
Comment (mbonabucya) [reply html/index.html]: @muneebulrehman 
This is good suggestion ,However it is not is the  instruction

File: html/index.html [hunk 5]
```diff
@@ -1,81 +1,334 @@
...
+                        <li>
+                            <p>Html</p>
+                           
+                        </li>
+                        <li>
+                            <p>css</p>
+                        </li>
+                        <li>
+                            <p>javascript</p>
+                        </li>
```
Comment (tresorsawasawa) [inline html/index.html:111]: - [ ] Kindly make sure for each card in the **works section** their content in styling have correct values as the ones provided from Figma, such as:
- **font-weight**, **font-size**, **line-height** on texts
- Each is supposed to have a border. Kindly add it and use the right color, size, and style.
- Use the **right spacing** between cards (e.g: **margins: top - right - left - bottom **
## Figma design  
card 
## Your design 
 s_card

File: css/style.css [hunk 6]
```diff
@@ -107,7 +115,359 @@
...
+  text-align: center;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-size: 17px;
+}
+
+.Projects button:hover {
+  border: 1px solid #396df2;
+  padding: 15px;
+}
```
Comment (tresorsawasawa) [inline css/style.css]: - [ ] Kindly make sure all your project buttons (**`See Project`** buttons and the   **`See Project`**) follow the button styling: **Active**, **Hover** and **Pressed** (see design). Check the right **colors** and **other properties** from Figma design on the similar image below:
buttons

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

##  Required Changes  :recycle::recycle: 
- [ ] **Check the comments under the review**

## Optional suggestions
- [ **OPTIONAL** ] For best practices, I suggest you remove the `index.html` file from the **html/** folder and put it in the **root directory**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @mbonabucya, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**